### ISSUE TITLE
Wanted labelled Dataset

### ISSUE THREAD

**Issue by shiv-acharii** [rfribeiro/mask-detector#2]
hey @rfribeiro 

I wanted your labeled dataset to train my TensorFlow object detection model, it would be helpful if you provide me
I see u have used LabelImg to manually do the annotation of bounding box containing images, so this would help me

**Comment by MohnishJain** [comment]
Yes @rfribeiro ,It will be helpful if you upload the pascal voc format labelled annotated xml files.

**Comment by xuan2261** [comment]
Dataset for trainning
https://github.com/X-zhangyang/Real-World-Masked-Face-Dataset

**Comment by xuan2261** [comment]
https://www.pyimagesearch.com/2020/05/04/covid-19-face-mask-detector-with-opencv-keras-tensorflow-and-deep-learning/#download-the-code

**Comment by rfribeiro** [comment]
hi guys,
Sorry but as I crawled them from internet and don't have the copyright of them
I choose to not share them

**Comment by shiv-acharii** [comment]
hi @rfribeiro 
Yes, I understand the concern. anyways amazing work done in the repo.

@xuan2261 
the dataset looks really good, but as @MohnishJain said we were actually looking for pascal VOC format labelled annotated XML files. since I am personally looking to implement it with Tensorflow Object Detection
